### PR TITLE
New package: python-progress-1.2

### PR DESCRIPTION
**Pull request by lafrenierejm**

### CODE REVIEW

File: srcpkgs/python-progress/template [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+build_style=python-module
+pycompile_module="progress"
+hostmakedepends="python-setuptools python3-setuptools"
+short_desc="Easy to use progress bars (Python2)"
+maintainer="Joseph LaFreniere <joseph@lafreniere.xyz>"
+homepage="https://github.com/verigak/progress"
+license="ISC"
+distfiles="${PYPI_SITE}/p/progress/progress-${version}.tar.gz"
+checksum=5b6a50f9ac76820f619d7789b5354881c8c10060a64047e415868d8f503cc2eb
+alternatives="progress:progress:/usr/bin/progress2"
```
Comment (Vaelatern) [inline srcpkgs/python-progress/template]: File does not exist. And should.

Comment (lafrenierejm) [inline srcpkgs/python-progress/template]: This is just a library, so it should not have anything in /usr/bin.  `alternatives` have been removed.

**Comment by lafrenierejm** [comment]
Force pushed.

**Comment by lafrenierejm** [comment]
Force pushed

File: srcpkgs/python-progress/template [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+short_desc="Easy to use progress bars (Python2)"
+maintainer="Joseph LaFreniere <joseph@lafreniere.xyz>"
+homepage="https://github.com/verigak/progress"
+license="ISC"
+distfiles="${PYPI_SITE}/p/progress/progress-${version}.tar.gz"
+checksum=5b6a50f9ac76820f619d7789b5354881c8c10060a64047e415868d8f503cc2eb
+
+python3-progress_package() {
+	noarch=yes
+	depends="python3-yaml"
```
Comment (Vaelatern) [inline srcpkgs/python-progress/template]: And not for py2?

Comment (lafrenierejm) [inline srcpkgs/python-progress/template]: The dependency should not have existed at all. Fixed in latest push.

**Pull request merged**